developer_resources/developer_resources.md

```diff
@@ -38,4 +38,24 @@ Follow these steps to generate rdocs for now and existing SDK gems:
 
 Rdoc options are controlled with this [config file](https://github.com/urbanopt/urbanopt-scenario-gem/blob/develop/.rdoc_options).
 
+## Developing with the CLI
+
+When developing locally on URBANopt core gems and testing new functionality and dependencies via the CLI, it may be necessary to install local versions of core gems.  
+
+For local development, you will want to set the environment variable *FAVOR_LOCAL_GEMS* to 1. This enables local copies of gems in the Gemfile.  There are 2 Gemfiles of interest:
```

Comment:
Can we add something like:

Remember that the only "falsey" things in Ruby are `nil` and `false`. Setting `FAVOR_LOCAL_GEMS==0` is still "truthy."

Reply:
If editing a gem that is already built, users will need to un-comment the appropriate block in the example project Gemfile.